tdnet/td/net/TcpListener.h

```diff
@@ -20,13 +20,14 @@ class TcpListener final : public Actor {
     virtual void accept(SocketFd fd) = 0;
   };
 
-  TcpListener(int port, ActorShared<Callback> callback);
+  TcpListener(int port, ActorShared<Callback> callback, char const * server_address = "0.0.0.0");
   void hangup() override;
 
  private:
   int port_;
   ServerSocketFd server_fd_;
   ActorShared<Callback> callback_;
+  char const * server_address_;
```

Comment:
Storing a `const char *` field is very unsafe. The type of the field needs to be replaced with `string`.